### DISCUSSION TITLE
latex header row height

### DISCUSSION THREAD

**Discussion by Liripo** [haozhu233/kableExtra#605] [Q&A]
![image](https://user-images.githubusercontent.com/54770415/114494753-76896f80-9c4f-11eb-8bc3-e44fb59dd451.png)
How to modify the height of the header in latex？ the header is too wide

**Comment by rcyost** [comment]
I think there is no way to edit the row height in kableExtra, you'd have to go through css in HTML or in pdf Latex try and inject Latex somehow? Currently trying to figure this out for latex table.
